app/api/util.py

```diff
@@ -0,0 +1,21 @@
+from django.conf import settings
+
+
+# returns the language codes from the django settings. ['de', 'en']
+def get_setting_lang_code():
```

Comment:
Please add a docstring, and change the name to plural, as it returns a list.

Docstrings can look like this:
```python
def get_setting_lang_codes():
    "Return the 2-character language codes from django settings"
    # ...
```